UI/src/view/navigationpanel/LinearAssetBox.js

```diff
@@ -51,8 +48,7 @@
     var element = $('<div class="panel-group simple-limit ' + className + 's"/>').append(elements.expanded).hide();
 
     function show() {
-      if (( (_.contains(userRoles, 'busStopMaintainer')) || (_.isEmpty(userRoles)) ) &&
-          !(_.contains(userRoles, 'operator') || _.contains(userRoles, 'premium'))) {
+      if (editModeToggle.hasNoRolesPermission(userRoles) || (_.contains(userRoles, 'premium') && elements.expanded.hasClass('panel maintenanceRoad')) ) {
```

Comment:
The user with the role 'premium' cannot perform modification on the asset 'serviceRoadMaintenance' because of that it was included this condition -> (_.contains(userRoles, 'premium') && elements.expanded.hasClass('panel maintenanceRoad')